browser/src/control/Control.UIManager.js

```diff
@@ -432,11 +432,13 @@ L.Control.UIManager = L.Control.extend({
 		var selectedTab = $('.ui-tab.notebookbar[aria-selected="true"]').attr('id') || 'Home-tab-label';
 		this.removeNotebookbarUI();
 		this.createNotebookbarControl(this.map.getDocType());
+		if (this._map._permission === 'edit') {
+			$('.main-nav').removeClass('readonly');
+		}
```

Comment:
should we also add class in opposite case?

Reply:
I thought about that. But I ended not doing because it seems control.Notebookbar.js:hideTabs is already doing that and control.Toolbar.js:onUpdatePermission as well (plus afraid to fix a inexistent problem and add regression :p )

Reply:
ok, then better to not touch this. At some point we should unify that so it will be done in one place.